src/components/Synthetics/TradeBox/TradeBox.tsx

```diff
@@ -1245,7 +1245,12 @@ export function TradeBox(p: Props) {
             </ToggleSwitch>
 
             {isLeverageEnabled && (
-              <LeverageSlider value={leverageOption} onChange={setLeverageOption} isPositive={isLong} />
+              <LeverageSlider
+                marks={[0.1, 1, 2, 5, 10, 15, 20, 25, 30, 40, 50]}
```

Comment:
Another small suggestion on the same field: would you like to consider moving the array to the constant out of the component body?

Reply:
Thanks, updated!